src/components/pages/Profile/History/ComputeJobs/Results.tsx

```diff
@@ -40,74 +41,81 @@ export default function Results({
   const [hasFetched, setHasFetched] = useState(false)
   const isFinished = job.dateFinished !== null
 
-  async function getResults() {
-    if (!account || !ocean || !job) return
+  useEffect(() => {
+    async function getResults() {
+      if (!account || !ocean || !job) return
+
+      try {
+        setIsLoading(true)
+        const computeService = ddo.findServiceByType('compute')
+        const jobStatus = await ocean.compute.status(
+          account,
+          job.did,
+          ddo,
+          computeService,
+          job.jobId
+        )
+        if (jobStatus) {
+          console.log('results', jobStatus)
```

Comment:
thought(non-blocking): console logs should be removed for production. But I guess in this particular case it is reasonable as it is considered software in development.